components/brave_wallet_ui/components/desktop/views/portfolio/portfolio-asset.tsx

```diff
@@ -295,6 +309,28 @@ export const PortfolioAsset = () => {
 
   const onNftDetailsLoad = React.useCallback(() => setNftIframeLoaded(true), [])
 
+  const onBridgeToAuroraButton = React.useCallback(() => {
+    if (bridgeToAuroraWarningShown) {
+      onOpenRainbowAppClick()
```

Comment:
`onOpenRainbowAppClick` used before it was defined

Reply:
@josheleonard Fixed here: 4665ea288ab16b67804ca03bf67df711af362a58